albumentations/augmentations/bbox_utils.py

```diff
@@ -363,10 +363,9 @@ def filter_bboxes(bboxes, rows, cols, min_area=0.0, min_visibility=0.0):
         transformed_box_area = calculate_bbox_area(bbox, rows, cols)
         bbox, tail = tuple(np.clip(bbox[:4], 0, 1.0)), tuple(bbox[4:])
         clipped_box_area = calculate_bbox_area(bbox, rows, cols)
-        if not transformed_box_area or clipped_box_area / transformed_box_area <= min_visibility:
+        if clipped_box_area == 0 or clipped_box_area / transformed_box_area < min_visibility:
```

Comment:
I think better to check `transformed_box_area == 0` because it is needed to avoid division by zero.

Reply:
I don't agree, I've changed it intentionally.
`transformed_box_area==0` => `clipped_box_area == 0`, but not `clipped_box_area == 0` => `transformed_box_area==0` .
If you check `transformed_box_area` and it is != 0 as before, then for `min_visibility=0` all boxes including invisible ones with `clipped_box_area == 0` will pass the filter. That's why we need check `clipped_box_area == 0` separately.
Of cause, we can check both just to demonstrate that we avoid division by 0, but it seems overkill :)

Reply:
Ok, I agree. But, I think, need to add comment that we check `clipped_box_area == 0` to properly process this  case, in future we will forget this detail.
Something like this:
```python
# Check `clipped_box_area == 0` to process case when `min_visibility=0`
# and strict comparison for case when `min_visibility=1`
if clipped_box_area == 0 or clipped_box_area / transformed_box_area < min_visibility:
```